tests/Routes/Wiki/SettingUpdateTest.php

```diff
@@ -71,6 +71,15 @@ public function provideValidSettings()
 
         $validProps = json_encode(['properties' => ['P31' => 'P1'], 'items' => ['Q1' => 'Q1']]);
         yield ['wikibaseManifestEquivEntities', $validProps, $validProps];
+
+        yield ['wwUseQuestyCaptcha', '1', '1'];
+        yield ['wwUseQuestyCaptcha', '0', '0'];
+        $validCaptchaQuestions = json_encode([
+            "How many vowels are in this question?" => ['12', 'twelve'],
+            "What is the chemical formula of water" => ['H2O'],
+            "2 + 4 = ?" => ['6', 'six']
+        ]);
+        yield ['wwCaptchaQuestions', $validCaptchaQuestions, $validCaptchaQuestions ];
```

Comment:
```suggestion
        yield ['wwCaptchaQuestions', $validCaptchaQuestions, $validCaptchaQuestions];
```

Reply:
should probably add some linting for this thing ;)